### ISSUE TITLE
Bug: Docs From Controllers Don't work when using inject()

### ISSUE THREAD

**Issue by HasanAshab** [ad-on-is/adonis-autoswagger#62]
**AdonisJS v6** supports controller injection.
When **inject()** decorator is used, docs from controller don't work.

Here is my partial controller
```typescript
@inject()
export default class AuthController {
  static readonly VERSION = 'v1'

  constructor(
    private readonly authService: AuthService,
    private readonly socialAuthService: SocialAuthService,
    private readonly twoFactorAuthService: TwoFactorAuthService
  ) {}
  
  /**
   * @logoutOnDevice
   * @summary Logout a user on specific device
   * @responseBody 200 - { "message": "Logged out successfully" }
   */
  async logoutOnDevice({ request, params, auth }: HttpContext) {
    await this.authService.logoutOnDevice(auth.user!, params.id)
    return 'Logged out successfully!'
  }
  //...
```

But when I just remove inject() everything works fine!

**Comment by ad-on-is** [comment]
There's an not yet documented param in the swagger config `debug: true`

 Would you mind testing that for me?

**Comment by HasanAshab** [comment]
@ad-on-is thanks for testing by yourself. I was sleeping :\. 

By the way, do you have any solution as the comments extracting lib is causing the error??

**Comment by HasanAshab** [comment]
It seems like the problem is not only **inject()** but any **decorator**.

also that lib don't work on that particular method, if we use **method decorator**

**Comment by ad-on-is** [comment]
I'll see what I can do

**Comment by ad-on-is** [comment]
Fixed in 3.1.0
